### ISSUE TITLE
[Explore Mode] Issues with explore mode on large worklists

### ISSUE THREAD

**Issue by jstanden** [jstanden/cerb#95]
A client tried to explore a list with 718,275 rows and it bogged down MySQL. We need to make this more efficient and implement realistic limits.

**Comment by jstanden** [comment]
The proper refactor would be something like this to avoid the looping in PHP at all:

~~~
SELECT 'hash1234', 
@rownum := @rownum + 1 AS pos, 
CONCAT('{"id":"',t.id,'","url":"','http:\\\/\\\/cerb.dev\\\/index.php\\\/profiles\\\/ticket\\\/',t.id,'\\\/conversation"}') 
FROM ticket t 
CROSS JOIN (SELECT @rownum := 0) AS counter 
WHERE 1 
ORDER BY t.created_date DESC;
~~~

**Comment by jstanden** [comment]
The quick fix for 7.3.7 will cap the explore results to 1,000 (4 pages of 250) for tickets + messages.  In 7.4+ we can refactor the whole approach.

**Comment by jstanden** [comment]
This hasn't been an issue for a while.
